src/odin/directionsbuilder.cc

```diff
@@ -101,23 +103,41 @@ TripDirections DirectionsBuilder::Build(const DirectionsOptions& directions_opti
 
 // Update the heading of ~0 length edges.
 void DirectionsBuilder::UpdateHeading(EnhancedTripPath* etp) {
+  auto is_walkway = [](TripPath_Use use) -> bool {
+    return ((use == TripPath_Use_kSidewalkUse) || (use == TripPath_Use_kFootwayUse) ||
```

Comment:
Should just be `return ((use >= TripPath_Use_kSidewalkUse && use <= TripPath_Use_kBridlewayUse));` ?

Reply:
thanks - i will update